nixos/tests/crater.nix

```diff
@@ -0,0 +1,41 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "crater";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ matthewcroughan onny ];
+  };
+
+  nodes = {
```

Comment:
We don't need to test firewall capabilities in this test as well. Just put everything on a single node and `curl` locally. Keep it simple.